rsconnect/actions.py

```diff
@@ -218,13 +218,13 @@ def test_server(connect_server):
     raise RSConnectException("\n".join(failures))
 
 
-def test_shinyapps_server(server: api.ShinyappsServer):
-    with api.ShinyappsClient(server) as client:
+def test_lucid_server(server: api.LucidServer):
```

Comment:
i'd probably leave this name (or call it `RStudioClient` (and `rstudio_server`)) since `lucid` is an internal code name and this is a public library. the name lucid doesn't mean anything to anyone using this.